### PR TITLE
ENT-7511 Skip update policy run with --skip-bootstrap-policy-run

### PR DESCRIPTION
**Pull request by craigcomstock**
Workaround for lmdb invalid argument issues in CI (ENT-7500)

Ticket: ENT-7511
Changelog: title

### CODE REVIEW

**Comment by craigcomstock** [comment]
@cf-bottom jenkins

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/7008/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/7008/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-7008/

File: libpromises/failsafe.cf [hunk 1]
```diff
@@ -383,7 +383,7 @@ bundle agent failsafe_cfe_internal_call_update
 
     # On Windows we need cf-execd to call update.cf, otherwise the daemons will
     # not run under the SYSTEM account.
-    !windows::
+    !windows.!skip_policy_on_bootstrap::
```
Comment (nickanderson) [inline libpromises/failsafe.cf:386]: I think this should be `!(windows|skip_policy_on_bootstrap)::` Not windows OR skip policy on bootstrap, the current logic reads not windows and not skip policy on bootstrap

Comment (craigcomstock) [reply libpromises/failsafe.cf:386]: Same logic. So a preference of how it reads?

https://csawesome.runestone.academy/runestone/books/published/csawesome/Unit3-If-Statements/topic-3-6-DeMorgan.html#de-morgan-s-laws

```
!(a || b) is equivalent to !a && !b
```

**Comment by craigcomstock** [comment]
retry CI after ubuntu 16 build slave is reconfigured to have i386 architecture, should pass with change in https://github.com/cfengine/masterfiles/pull/2053

[Build Status](https://ci.cfengine.com/job/pr-pipeline/7011/)

File: libpromises/failsafe.cf (hunk 1 above)
Comment (nickanderson) [reply libpromises/failsafe.cf:386]: bah, ignore me.

yes, i do have a preference for (). I get tripped up easily.

**Pull request merged**